Faultify.Cli/Program.cs

```diff
@@ -142,7 +144,7 @@ private async Task Run(Settings settings)
         {
             ITestHostRunFactory testHost = settings.TestHost switch
             {
-                _ => new DotnetTestHostRunnerFactory() // TODO: Use Faultify.TestRunner.XUnit/NUnit for in memory testing. 
+                _ => new NUnitTestHostRunnerFactory() // TODO: Use Faultify.TestRunner.XUnit/NUnit for in memory testing. 
```

Comment:
I understand NUnit is the focus for now. Let's keep in mind this is enabled currently, only when XUNIT is working we can do a release. Ideally, we want dotnet test to remain working. Does it work currently? Perhaps we can put it as default sucht that faultify works for everyone using the master branch version of faultify.